### PR TITLE
Parse BCID cards

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[ID Data does not parse British Columbia bar codes](https://trello.com/c/PLlgeQ9j/536-id-data-does-not-parse-british-columbia-bar-codes)

**Review by csrapr — APPROVED** [review]

**Review by csrapr — CHANGES_REQUESTED** [review]
The person in the example code weighs 95 pounds, but what if they weighed 195 pounds?
I tested the code and the output is this: `weight: "19"` (notice how it is ignoring the trailing 5)
Also if the weight is 195 the eye color becomes "NBL"

try testing here:
https://npm.runkit.com/lodash
https://paste.ofcode.org/32KtvRjtPtrJhtCFdkBGA3f

**Review by csrapr — APPROVED** [review]

File: lib/compliance/parsepdf417.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const crypto = require('crypto')
 const _ = require('lodash/fp')
 
 const parsers = [
+  require('./parsepdf417-ca-bc'),
```
Comment (RafaelTaranto) [inline lib/compliance/parsepdf417.js:8]: isn't it better to check for us first?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

File: lib/compliance/parsepdf417-ca-bc.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  const track2Start = data.indexOf(';')
+  const track3Start = data.indexOf('_%')
+
+  if (track1Start === -1 || !track2Start === -1 || !track3Start === -1) return null
+
+  return parseRaw(data)
+}
+
+function parseRaw (data) {
+  const track1Start = data.indexOf('%')
```
Comment (joshmh) [inline lib/compliance/parsepdf417-ca-bc.js:14]: We shouldn't do these searches twice. If we merge parse and parseRaw, the combined function is just one line longer than parseRaw (the ``if`` statement).

**Review by joshmh — CHANGES_REQUESTED** [review]

**Pull request merged**